apps/cookbook/src/app/examples/modal-example/modal-example-outlet.component.ts

```diff
@@ -1,48 +1,51 @@
 import { Component } from '@angular/core';
 import { Params } from '@angular/router';
 
-import { AlertConfig, ModalController } from '@kirbydesign/designsystem';
+import { ModalController } from '@kirbydesign/designsystem';
 
 const config = {
   selector: 'cookbook-modal-example-outlet',
   template: `<button kirby-button (click)="navigateToModalRoute('page1', {awesomeQueryParam: 'awesome value'})">Open modal by route</button>
-<button kirby-button (click)="navigateToModalRoute('page1', {awesomeQueryParam: 'awesome value'}, alertConfig)">Open modal by route with alert</button>
-<button kirby-button kirbyModalRouterLink="page1" [kirbyModalQueryParams]="{awesomeQueryParam: 'awesome value'}">Open modal by router link</button>
-<button kirby-button kirbyModalRouterLink="page1" [kirbyModalQueryParams]="{awesomeQueryParam: 'awesome value'}" [kirbyAlertConfig]="alertConfig">Open modal by router link with alert</button>
-<button kirby-button class="deeplink" (click)="navigateToModalRoute(['/examples', 'modal', 'page1'], {awesomeQueryParam: 'awesome value'})">Deep link to modal route</button>
-<button kirby-button class="deeplink" [kirbyModalRouterLink]="['/examples', 'modal', 'page1']" [kirbyModalQueryParams]="{awesomeQueryParam: 'awesome value'}">Deep link to modal by router link</button>
-<button kirby-button class="deeplink" [kirbyModalRouterLink]="['/examples', 'modal-route-with-url-param', '1978', 'page1']">Deep link to modal with url param</button>`,
+<button kirby-button kirbyModalRouterLink="page1" [kirbyModalQueryParams]="{awesomeQueryParam: 'awesome value'}">Open modal by router link</button>`,
   defaultCodeSnippet: `
-  readonly alertConfig: AlertConfig = {
-    title: 'Do you want to close the modal?',
-    okBtn: {
-      text: 'Yes',
-      isDestructive: true,
-    },
-    cancelBtn: 'No',
-  };
-  
   constructor(private modalController: ModalController) {}
 
-  navigateToModalRoute(path: string | string[], queryParams?: Params, alertConfig?: AlertConfig) {
-    this.modalController.navigateToModal(path,  queryParams, alertConfig);
+  navigateToModalRoute(path: string | string[], queryParams?: Params) {
+    this.modalController.navigateToModal(path,  queryParams);
   }`,
-  modalRouteCodeSnippet: `{
+  modalRouteCodeSnippet: `export const routes: ModalEnabledRoutes = [
+  {
     path: 'main-route-presented-behind-the-modal',
     component: SomeComponent,
     children: [
       {
         path: 'child-route-presented-in-modal',
         outlet: 'modal',
         component: FirstChildComponent,
+        
+        // optional modalConfig passed via the Angular-router's built in data object
```

Comment:
Consider wording/casing:
```suggestion
        // optional modalConfig passed via Angular Router's built in data object
```